### ISSUE TITLE
' Only you can see this message' output

### ISSUE THREAD

**Issue by dofu** [sofetch/slack-standup-bot#37]
Hi,

May I know why do i get this? Even when the bot is working.

``` 
slackbot [4:59 PM] Only you can see this message
<DOCTYPE html>
<html lang='en'>
<head>
<title>Standup Bot by So Fetch</title>
<meta charset='utf-8'>
<meta content='IE=edge' http-equiv='X-UA-Compatible'>
<meta content='width=device-width, initial-scale=1' name='viewport'>
<link rel="stylesheet" media="all" href="/assets/application-9826fd0becbbc2572e64787dc5b3b0e6cc68dad928ba2704ec8d018837bd18ef.css" />
<script src="/assets/application-68186a1686708ecedd55f9bb5712d4780c825c0e7355faa0b994bf20bfd378bd.js" data-turbolinks-track="true"></script>
<meta name="csrf-param" content="authenticity_token" />
<meta name="csrf-token" content="7JQ6S3IFNkKnpqnTPjGsykiuElvBMj338yPVWXJVLvf4ZSPcGf+hVKs6bXwbk0zVCWCpa6TM08jNCxL7Uh+xCA==" />
<link href='https://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,500,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
</head>
</html>
<body>
<div class='wrapper'>
<div class='colors-3'>
<nav class='navbar navbar-fixed-top'>
<div class='container'>
<div class='navbar-brand'>
<img src="/assets/standup_bot_logo-6b03627016edc14e04354a16d50702e1354f8e8783b8b65050b71fb15d0fb35d.png" alt="Standup bot logo 6b03627016edc14e04354a16d50702e1354f8e8783b8b65050b71fb15d0fb35d" />
</div>
</div>
</nav>
<div class='container main-content'>
<div class='alert alert-success'>
<b>Well done!</b>
Everything is ok, the standup will start soon.
</div>

</div>
</div>
</div>
<footer class='footer container'>
<p class='text-muted'>
Made by
<a href="http://www.sofetch.io">So Fetch.</a>
<img src="/assets/sf_logo_standup_bot-ce528b85fea8e439a3eb6047dc5ad20ed15178cee4b07034288a0b8333f8dd82.png" alt="Sf logo standup bot ce528b85fea8e439a3eb6047dc5ad20ed15178cee4b07034288a0b8333f8dd82" />
</p>
</footer>
</body> ```

**Comment by santiagodoldan** [comment]
@dofu that's because of some changes I did the last week, right now the /api/start endpoint returns html to show errors related to the settings. For example if you enter a wrong bot username, that page will show you a message that says "There is no Bot called @your_bot within the #your_channel Channel".

I would need to create a different endpoint for the slack command to avoid showing those errors, or maybe return a string with the errors, I'll let you know when that's ready in master.

You can start the standup visiting http://yourdomainapp.com/api/start and then you won't receive that slack message.

Cheers

**Comment by santiagodoldan** [comment]
@dofu I created a new PR to fix this weird behaviour #39, I'll let you know as soon as that PR is merged into master.

**Comment by santiagodoldan** [comment]
@dofu the PR was merged into master, you'll need to change your slash command, add ?format=text at the end of the url, the README is updated for more information
